lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
```

Comment:
The `latest` tag in docker essentially<sup>1</sup> means the latest release, in this case `1.5.0`. Think semver's `*`.

<sup>1 - you can make the `latest` tag point to whatever version you want</sup>